HousingRepairsOnlineApi/Helpers/AppointmentConfirmationSender.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Threading.Tasks;
+using HousingRepairsOnlineApi.Domain;
+using HousingRepairsOnlineApi.UseCases;
+
+namespace HousingRepairsOnlineApi.Helpers
+{
+    public class AppointmentConfirmationSender : IAppointmentConfirmationSender
+    {
+        private readonly ISendAppointmentConfirmationEmailUseCase sendAppointmentConfirmationEmailUseCase;
+        private readonly ISendAppointmentConfirmationSmsUseCase sendAppointmentConfirmationSmsUseCase;
+
+        public AppointmentConfirmationSender(ISendAppointmentConfirmationEmailUseCase sendAppointmentConfirmationEmailUseCase, ISendAppointmentConfirmationSmsUseCase sendAppointmentConfirmationSmsUseCase)
+        {
+            this.sendAppointmentConfirmationEmailUseCase = sendAppointmentConfirmationEmailUseCase;
+            this.sendAppointmentConfirmationSmsUseCase = sendAppointmentConfirmationSmsUseCase;
+        }
+
+        public async Task Execute(RepairRequest repairRequest, string repairId)
+        {
+            switch (repairRequest?.ContactDetails?.Type)
+            {
+                case Constants.Email:
+                    sendAppointmentConfirmationEmailUseCase.Execute(repairRequest.ContactDetails.Value, repairId, repairRequest.Time.Display);
+                    break;
+                case Constants.Sms:
+                    sendAppointmentConfirmationSmsUseCase.Execute(repairRequest.ContactDetails.Value, repairId,
+                        repairRequest.Time.Display);
+                    break;
+                default:
+                    throw new Exception("Invalid contact type");
```

Comment:
It would be best to provide the invalid contact type such that it's shown in error reporting (logs, etc) making it easier to find the source and possibly to fix too.
```suggestion
                    throw new Exception($"Invalid contact type: {repairRequest?.ContactDetails?.Type}");
```